### DISCUSSION TITLE
Resolve platform IP's to GSA subdomain.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#469] [Suggestions]
It would be great if we had something like platform.gameserverapp.com that resolved to the platform IP's as listed here. https://help.gameserverapp.com/article/238-platform-ips

As an example take a look at all.sipis.acrobits.cz 

https://imgur.com/a/YLafNE4 it has multiple A records.

Edit: This would be so that firewalls can auto update when GSA changes servers or the like.

**Comment by mvaessen** [comment]
Done! 

DNS record: platform.gameserverapp.com
